demos/nextjs-ai/lib/langgraph.ts

```diff
@@ -0,0 +1,5 @@
+import { Client } from "@langchain/langgraph-sdk";
+
+export const langgraph = new Client({
+  apiUrl: process.env.LANGGRAPH_API_URL || "http://localhost:54367",
```

Comment:
Is having a LangGraph server a requirement to use FCAT with LangChain? it feels quite complex IMO. Is there no other way to handle interrupts at least in a NextJS app maybe using the Vercel AI SDK for that part?